metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -130,6 +130,17 @@ object ServerCommands {
        |""".stripMargin
   )
 
+  val NewScalaWorksheet = new Command(
+    "new-scala-worksheet",
+    "Create new scala worksheet",
+    """|Create and open new scala worksheet file,
```

Comment:
Nitpick, maybe without the comma and in one line?

```
    """|Create and open new scala worksheet file at the argument location with the argument + ".worksheet.sc" name.
```

Reply:
:+1: